app/assets/javascripts/views/story_view.js

```diff
@@ -74,6 +74,20 @@ module.exports = FormView.extend({
 
     // Set up CSS classes for the view
     this.setClassName();
+
+    this.attachinaryOptions = {}
+    this.fetchAttachinaryOptions();
+  },
+
+  fetchAttachinaryOptions: function() {
+    attOptions = new AttachmentOptions({
+      refreshCallback: this.setAttachinaryOptions.bind(this)
+    });
+    attOptions.fetch();
+  },
+
+  setAttachinaryOptions: function(options) {
+    this.attachinaryOptions = options;
```

Comment:
Will this trigger a re-render of the component that receive the options as props?